index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+	<link rel="stylesheet" type="text/css" href="./css/index.css">
+	<title>NVIDIA GTC 2022 KEYNOTE</title>
+</head>
+<body>
+  <div id="background">
+   <header>
+    <nav id="mobile-nav">
+    	<i id="menu-btn" class="fas fa-bars fa-2x"></i>
+      <ul id="mobile-list" class="dn">
+      	<li class="mobile-list-item"><a href="./">Home</a></li>
+      	<li class="mobile-list-item"><a href="./about.html">About</a></li>
+      	<li class="mobile-list-item"><a href="./#the-program">Program</a></li>
+      	<li class="mobile-list-item"><a href="./#speakers">Speakers</a></li>
+      	<li class="mobile-list-item"><a href="./about.html#partners">Sponsors</a></li>
+        <li class="mobile-list-item"><a href="#">Buy tickets now</a></li>
+      </ul>	
+    </nav>
+   <nav id="desktop-nav">
+    	<ul id="social-personal-links">
+    		<li><i class="fab fa-facebook-f"></i></li>
+    		<li><i class="fab fa-twitter"></i></li>
+    		<li>English</li>
+    		<li>My Page</li>
+    		<li>logout</li>
+    	</ul>	
+      <ul id="sections-pages-links">
+      	<li>
+      		<a href="./">
+      		  <img alt="logo" src="./images/general/event-logo.png" />
+      		</a>
+      	</li>
+      	<li><a href="./about.html">About</a></li>
+      	<li><a href="./#the-program">Program</a></li>
+      	<li><a href="./#speakers">Speakers</a></li>
+      	<li><a href="./#partners">Sponsors</a></li>
+        <li><a href="#">Buy tickets now</a></li>
+      </ul>	
+    </nav>	
+   </header>
+   <section id="headline">
+  	 <p id="welcome-sentence">"Hello! Sharing world"</p>
+  	 <h1 id="Heading">The World’s Brightest Minds. One Unique Event 2022.</h1>
+  	 <p id="introduction">
+	   NVIDIA GTC is more than a game-changing AI developer conference. It’s a global community committed to decoding the world’s greatest challenges, transforming every major industry workflow, and exploring tomorrow’s next big ideas—together. Join us this March and discover how to accelerate your life's work.
+  	</p>
```

Comment:
- [ ] [OPTIONAL] Please notice that with this introduction text section the text is breaking out of its container please address this issue.
![image](https://user-images.githubusercontent.com/18730106/152519396-bde35c03-ac46-404a-a711-3f41c9d6e2f8.png)